analyzer/testdata/src/dgryski/learngo.go

```diff
@@ -38,8 +38,7 @@ func main() {
 // Functions have parameters in parentheses.
 // If there are no parameters, empty parentheses are still required.
 func beyondHello() {
-	var x int // Variable declaration. Variables must be declared before use.
-	x = 3     // Variable assignment.
+	var x int = 3 // Variable assignment.
```

Comment:
AFAIK, this is test data for unit test purpose. I.e any file under the testdata/src directory is intentionally written with issues that are supposed to be caught by the analyzer.